Add a glossary (#27)
project-chip/matter-handbook: content/Glossary/_index.md

+++
title = "Glossary"
chapter = true
weight = 6
pre = "<b>6. </b>"
+++

> # Glossary

Reviewer: ```suggestion
# Glossary & Acronym Decoder
```

| ACL      | Access Control List. This is a list of entries in the Access Control cluster that grant access privileges to device and groups                                            |
| adoc     | The filename suffix for asciidoc, which is the markup language we use to write the specification and test plans. [asciidoc](https://asciidoc.org/)                        |
| Alliance | Refers to the Connectivity Standards Alliance (CSA)                                                                                                                       |
| asciidoc | Markup language we use to write the specification and test plans. [asciidoc](https://asciidoc.org/)                                                                       |
| Causeway | Site used to store documents and calendars and track group memberships etc. [https://groups.csa-iot.org/](https://groups.csa-iot.org/)                                    |
| CASE     | Certificate Authenticated Session Establishment. This is the session establishment mechanism used during normal Matter operations                                         |

> | CD       | Certification declaration - this declaration is provided by devices to prove they have passed certification                                                               |

Reviewer: ```suggestion
| CD       | Certification Declaration - this declaration is provided by devices to prove they have passed certification                                                               |
```

> | CHIP     | Matter used to be called CHIP (connected home over IP) before it got its current name. Much of the SDK still uses that prefix.                                            |

Reviewer: ```suggestion
| CHIP     | Matter used to be called Project CHIP (Connected Home over IP) before it got its current name. Much of the SDK still uses that prefix.                                            |
```

> | CSA      | Connectivity standards alliance - the umbrella organization for Matter                                                                                                    |

Reviewer: ```suggestion
| CSA      | Connectivity Standards Alliance - the member-driven organization that manages the Matter standard                                                                                                   |
```

> | CSG      | Certification subgroup [CSG](https://groups.csa-iot.org/wg/matter-csg/workgroup)                                                                                          |

Reviewer: ```suggestion
| CSG      | Certification Sub Group [CSG](https://groups.csa-iot.org/wg/matter-csg/workgroup)                                                                                          |
```

> | DAC      | Device attestation certificates. See [Attestation](https://handbook.buildwithmatter.com/howitworks/attestation/)                                                          |

Reviewer: ```suggestion
| DAC      | Device Attestation Certificates. See [Attestation](https://handbook.buildwithmatter.com/howitworks/attestation/)                                                          |
```

| DM       | Data Model. The data model describes the organization of data within a cluster. DM is sometimes also used to refer to the DMTT.                                           |
| DMTT     | Data Model Tiger Team.                                                                                                                                                    |
| DNS-SD   | Domain name service (DNS) service discovery. This is the mechanism used to locate Matter devices on the IP network                                                        |

> | DRLC     | Demand Load Response Control                                                                                                                                              |

Reviewer 2: ```suggestion
| DRLC     | Demand Response Load Control                                                                                                                                              |
```

| EVSE     | Electric Vehicle Supply Equipment                                                                                                                                         |
| ICAC     | Intermediate Certificate Authority Certificate See [Operational Credentials](https://handbook.buildwithmatter.com/howitworks/fabric/)                                     |
| ICD      | Intermittently Connected Devices. These device can sleep for relatively long periods and will be unreachable over Matter during these times.                              |
| mDNS     | Multicast DNS - the mechanism used to perform DNS-SD over Wi-Fi                                                                                                           |

> | MPSG     | Marketing and product subgroup [MPSG](https://groups.csa-iot.org/wg/matter-mpsg/workgroup)                                                                                |

Reviewer: ```suggestion
| MPSG     | Marketing and Product Sub Group [MPSG](https://groups.csa-iot.org/wg/matter-mpsg/workgroup)                                                                                |
```

| NOC      | Node operational certificate. See [Operational Credentials](https://handbook.buildwithmatter.com/howitworks/fabric/)                                                      |
| NOSCR    | Node Operational Certificate Signing Request. See [Attestation](https://handbook.buildwithmatter.com/howitworks/attestation/)                                             |

> | PAA      | Product attestation authority. Root certificate for device attestation. See [Attestation](https://handbook.buildwithmatter.com/howitworks/attestation/)                   |

Reviewer: ```suggestion
| PAA      | Product Attestation Authority. Root certificate for device attestation. See [Attestation](https://handbook.buildwithmatter.com/howitworks/attestation/)                   |
```

| PID      | Product identifier                                                                                                                                                        |
| PIXIT    | Protocol Implementation eXtra Information for Testing. These are configuration values provided to certification tests.                                                    |
| RCAC     | Root Certificate Authority Certificate. See [Operational Credentials](https://handbook.buildwithmatter.com/howitworks/fabric/)                                            |
| SDK      | Software development kit - a software package implementing the specification [github] (https://github.com/project-chip/connectedhomeip)                                   |
| spec     | Short for specification. [github] (https://github.com/CHIP-Specifications/connectedhomeip-spec)                                                                           |
| SRP      | Service Registration Protocol. This is the mechanism used to register thread devices for DNS-SD. Border routers implement this protocol to allow Thread device discovery. |

> | SVE      | Specification validation event - this follows the test events and acts as the final check on the certification package for a release.                                     |

Reviewer: ```suggestion
| SVE      | Specification Validation Event - this follows the test events and acts as the final check on the certification package for a release.                                     |
```

| SWTT     | [Software development tiger team](https://groups.csa-iot.org/wg/matter-tsg-swd/workgroup). This is the group of people working on the SDK.                                |

> | TE       | Test event. These events are held during development to verify that test script and SDK development are on track.                                                         |

Reviewer: ```suggestion
| TE       | Test Event. These events are held during development to verify that test scripts and SDK development are on track.                                                         |
```

> | TSG      | Technical subgroup [TSG](https://groups.csa-iot.org/wg/matter-tsg/workgroup)                                                                                              |

Reviewer: ```suggestion
| TSG      | Technical Sub Group [TSG](https://groups.csa-iot.org/wg/matter-tsg/workgroup)                                                                                              |
```

| TT       | Tiger team - a small group focused on one activity or feature.                                                                                                            |
| VID      | Vendor identifier                                                                                                                                                         |

> | WG       | Working group. [Matter](https://groups.csa-iot.org/wg/matter-wg/workgroup) is a working group in the CSA.                                                                 |

Reviewer: ```suggestion
| WG       | Work Group. [Matter](https://groups.csa-iot.org/wg/matter-wg/workgroup) is a working group in the CSA.                                                                 |
```
